Assets/Scripts/Potions/Potion.cs

```diff
@@ -28,10 +37,13 @@ public List<CollectablePlantsEnum> PlantsInPotion
         set { this.plantsInPotion = value; }
     }
 
+    // Constructor
     public Potion(string potionName, int potionCoinValue, List<CollectablePlantsEnum> plantsInPotionRecipe)
     {
+        random = new System.Random();
         PotionName = potionName;
         PotionCoinValue = potionCoinValue;
         PlantsInPotion = plantsInPotionRecipe;
+        potionImage = Resources.Load<Sprite>("Potion" + random.Next(1,3).ToString());
```

Comment:
Why are we loading a random potion? Don't we look at the plants in the potion to decide what potion it is?

Reply:
I can have 1 image per potion type is that's prefered.

Reply:
It's up to you, but consistency might help for the same potion to avoid confusion. Does anyone else have any input?